sarif.json

```diff
@@ -0,0 +1,70 @@
+{
```

Comment:
Did you mean to include this file @pankhurisaxena28? It seems unused in this PR.

Reply:
This was generated automatically as a result of `npm build` and `npm run test`.

Reply:
Nonetheless, we do not want this file to be present in the repository. Therefore, I have deleted the file.

Reply:
We may want to add it to a `.gitignore` then in the future.